doc/xml/reference.xml

```diff
@@ -744,6 +744,15 @@
                         <example>5G</example>
                     </config-key>
 
+                    <!-- CONFIG - BACKUP SECTION - EXPIRE-AUTO -->
+                    <config-key id="expire-auto" name="Expire Auto">
+                        <summary>Automatically run the <cmd>expire</cmd> command after a successful backup.</summary>
+
+                        <text>The setting is enabled by default. Don't forget to manually run the <cmd>expire</cmd> command if you disable it.</text>
```

Comment:
Instead of 
`Don't forget to manually run the <cmd>expire</cmd> command if you disable it.` 

I suggest: 
`Use caution when disabling this option; doing so will result in retaining all backups and archives indefinitely which could cause your repository to run out of space.`

Reply:
Hi,
Thanks for the review and the suggestion. It has been applied.